test/Microsoft.Dnx.DesignTimeHost.FunctionalTests/DthCompilationTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Dnx.CommonTestUtils;
+using Microsoft.Dnx.DesignTimeHost.FunctionalTests.Infrastructure;
+using Microsoft.Dnx.Runtime;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Microsoft.Dnx.DesignTimeHost.FunctionalTests
+{
+    [Collection(nameof(DthFunctionalTestCollection))]
+    public class DthCompilationTests
+    {
+        private readonly DthFunctionalTestFixture _fixture;
+
+        public DthCompilationTests(DthFunctionalTestFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        public static IEnumerable<object[]> ClrRuntimeComponents
+        {
+            get { return TestUtils.GetClrRuntimeComponents(); }
+        }
+
+        [Theory]
+        [MemberData(nameof(ClrRuntimeComponents))]
```

Comment:
As far as I know we ran DTH with CLR in VS and VS Code today. So I limit the scope. The project is still compiled in multiple frameworks including CoreCLR.